scripts/load_data_into_db.py

```diff
@@ -0,0 +1,99 @@
+import getpass
+import pandas as pd
+import psycopg2
+import sys
+
+def create_table(hostname, dbname, user):
+    commands = [
+            """
+            DROP TABLE IF EXISTS service_providers
+            """,
+            """
+            CREATE TABLE service_providers (
+                id INTEGER PRIMARY KEY,
+                census_division_id INTEGER,
```

Comment:
Mark `census_division_id` as a FOREIGN KEY similar to how the demographics tables does it.